### ISSUE TITLE
VRCfury prefab uses 194 memory while modular avatar uses 144 memory?

### ISSUE THREAD

**Issue by LogMeIn-Hamachi** [Dyslex1k/VRC-Skeletal-Hands#4]
I have 111 used memory out of 256 available. 

Adding VRCfury prefab and trying upload results in it using 304 memory

Using modular avatar prefab uses 144 memory and upload succeed.

i want to try vrcfury prefab because osc smooth does not work on modular avatar prefab on the (1.0) release. 

But to troubleshoot i tried updating prefab to (1.1) without osc smooth to see if memory usage in parameters was solved but it still uses 194 memory on vrcfury prefab while modular avatar uses the expected 144.

First i thought i had misunderstood difference between bits and memory in parameters. But it does not make sense that Modular avatar would use the expected bits while vrcfury does not.

**Comment by LogMeIn-Hamachi** [comment]
Okay comparing Modular avatar parameters and vrcfury parameters theres some differences.

Vrcfury have all parameters set too "Synced"

While modular avatar have "Local only" and "Saved" on following: SH/Controls/Left/CurlOn , SH/Controls/Left/SplayOn , SH/Controls/Right/CurlOn, SH/Controls/Right/SplayOn, SH/Controls/Left/On, SH/Controls/Right/On.

Changing those in vrcfury too "Saved" and not "Synced" results in 146 memory used instead.

Have not tested ingame yet tho.

**Comment by Dyslex1k** [comment]
That probably was an error on my end. I have been spending time today working on making better animators and lowering the Bit count. I'll look into this myself in a little bit!

SH/Controls/Left/CurlOn , SH/Controls/Left/SplayOn , SH/Controls/Right/CurlOn, SH/Controls/Right/SplayOn, SH/Controls/Left/On, SH/Controls/Right/On.

These parameters should not be synced and i will update the package so that they are not!

**Comment by Dyslex1k** [comment]
Yeah my bad the parameters are set incorrectly on the parameter set pushing an update now.

It unfortunately has to use 2 more bits just to track if the animation is running or not so the total will be 146.

**Comment by LogMeIn-Hamachi** [comment]
Thank you allot!

I saw in patch notes that you removed OSC smooth on 1.1 due too issues. But I have never gotten OSC smooth to work on 1.0 with vrcfury (with or without adding smoothed parameters in advanced tab) or modular avatar prefab.

I tested by launching 2 Vrchat and joining same instance on EU server and fingers looked like the 10hz update with laggy movement. 

But since you already said adding it back in the future is planned I guess best thing is to wait for that.
